onnxruntime/core/optimizer/bias_softmax_fusion.cc

```diff
@@ -61,7 +61,7 @@ bool TryBiasSoftmaxSubgraphMatch(Graph& graph, Node& start, Node*& add, Node*& s
 
   // check add is only consumed by softmax with matching exec provider
   Node& softmax_node = *graph.GetNode(add_node.OutputNodesBegin()->Index());
-  if (!graph_utils::IsSupportedOptypeVersionAndDomain(softmax_node, "Softmax", {1, 11}) ||
+  if (!graph_utils::IsSupportedOptypeVersionAndDomain(softmax_node, "Softmax", {1, 11, 13}) ||
```

Comment:
I dont think this is so straightforward. There were multiple changes for this op in version 13.
1. op changed from primitive op to function - should be OK
2. a new bfloat16 type was added - should be OK
3. default value for axis attr changed - more changes are needed to accommodate this change

Please also add relevant tests

Reply:
will check again, thx

Reply:
regarding bfloat16 type addition - initially I said it should be OK... but please check whether the fused op supports this type... if not then you will need a check around this as well .i.e. to reject fusion if original op has bfloat16 type

Reply:
regarding bfloat16 type addition - initially I said it should be OK... but please check whether the fused op supports this type... if not then you will need a check around this as well .i.e. to reject fusion if original op has bfloat16 type

Reply:
You are right, bf16 has not been supported by FusedBiasSoftmax yet, will add type check around original op.  
Good catch.